app/assets/javascripts/modules/track-brexit-qa-choices.js

```diff
@@ -0,0 +1,40 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (global, GOVUK) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  GOVUK.Modules.TrackBrexitQaChoices = function () {
+    this.start = function (element) {
+      track(element)
+    }
+
+    function track (element) {
+      element.on('submit', function (event) {
+        var $checkedOption, eventLabel, options
+        var $submittedForm = $(event.target)
+        var $checkedOptions = $submittedForm.find('input:checked')
+        var questionKey = $submittedForm.data('question-key')
+
+        if ($checkedOptions.length) {
+          $checkedOptions.each(function (index) {
+            $checkedOption = $(this)
+            eventLabel = $checkedOption[0].labels.length ?
+              $($checkedOption[0].labels[0]).text().trim()
+              : $checkedOption.attr('name').replace('[]', '')
+            options = { transport: 'beacon', label: eventLabel }
+
+            GOVUK.SearchAnalytics.trackEvent('brexit-checker-qa', questionKey, options)
```

Comment:
This is mostly a lack of understanding on my part: it looks like we're sending three strings to GA - 'brexit-checker-qa', '<questionKey>' and '<eventLabel>' - but I think GA only has two fields to store them (category and label). Do you know what happens to the other string?

Reply:
It gets passed in as action:

<img width="668" alt="Screen Shot 2019-09-06 at 14 18 11" src="https://user-images.githubusercontent.com/4599889/64430917-3811f300-d0b1-11e9-9505-bcbc8ace16d7.png">

Also in JS, you can pass any number of arguments to a function and it can be accessed via the `arguements` variable.

<img width="654" alt="Screen Shot 2019-09-06 at 14 21 32" src="https://user-images.githubusercontent.com/4599889/64431327-1c5b1c80-d0b2-11e9-8322-54f87274da80.png">
<img width="616" alt="Screen Shot 2019-09-06 at 14 21 24" src="https://user-images.githubusercontent.com/4599889/64431333-1f560d00-d0b2-11e9-8904-ef5c14d528ce.png">

Try it out:
`var x = function(){console.log(arguments);}`

<img width="675" alt="Screen Shot 2019-09-06 at 14 22 34" src="https://user-images.githubusercontent.com/4599889/64431372-31d04680-d0b2-11e9-9949-34c2d553667e.png">

Reply:
Ah yes - forgot all about action!